### PR TITLE
data manager tests part 3

### PR DESCRIPTION
**Pull request by Cadair**
The tests still were not getting collected when run with tox.

### CODE REVIEW

**Comment by nabobalis** [comment]
WE MISSED THE INIT FILE?

**Comment by nabobalis** [comment]
```
        else:
>           time = parse_time(details['time']).datetime
E           KeyError: 'time'
```

**Comment by Cadair** [comment]
The windows fail is real.

**Comment by nabobalis** [comment]
I'll have a look.

**Comment by nabobalis** [comment]
Online failure:
```
208     >>> from sunpy.net.helio import parser
209     >>> parser.wsdl_retriever()  # doctest: +REMOTE_DATA
UNEXPECTED EXCEPTION: ValueError('No online HELIO servers can be found.')
Traceback (most recent call last):

  File "/opt/hostedtoolcache/Python/3.8.0/x64/lib/python3.8/doctest.py", line 1328, in __run
    exec(compile(example.source, filename, "single",

  File "<doctest sunpy.net.helio.parser.wsdl_retriever[1]>", line 1, in <module>

  File "/home/vsts/work/1/s/.tox/py38-online/lib/python3.8/site-packages/sunpy/net/helio/parser.py", line 229, in wsdl_retriever
    fail()

  File "/home/vsts/work/1/s/.tox/py38-online/lib/python3.8/site-packages/sunpy/net/helio/parser.py", line 220, in fail
    def fail(): raise ValueError("No online HELIO servers can be found.")
```

File: sunpy/data/data_manager/cache.py [hunk 1]
```diff
@@ -107,7 +108,14 @@ def _has_expired(self, details):
         `bool`
             Whether the url has expired or not.
         """
-        return self._expiry and datetime.now() - datetime.fromisoformat(details['time']) > self._expiry
+        time = details.get("time", datetime.now().isoformat())
+
+        # TODO: Remove this once we depend on Python >=3.7
```
Comment (vn-ki) [inline sunpy/data/data_manager/cache.py]: What's this magic?

Comment (nabobalis) [reply sunpy/data/data_manager/cache.py]: What bit?

Comment (vn-ki) [reply sunpy/data/data_manager/cache.py]: Is fromisoformat not present in some python versions? I thought that was always there. Interesting.

Comment (nabobalis) [reply sunpy/data/data_manager/cache.py]: It was added in 3.7.

**Pull request merged**